node-db/week-13/lesson.md

```diff
@@ -113,14 +114,54 @@ Node.js. It simplifies base features of Node.js, making it easier and faster to
 build your application's backend. Learning Express gives you a great foundation
 for becoming a Node.js developer.
 
-In the next steps, we will building a **CMS (Content Management System)** - a
-system to write, publish and save blog posts. Even though we could use core
-libraries like `http` to build this system, this becomes very complicated and
-non-maintainable very quickly. That is why we use frameworks like `Express`.
+# APIs
+
+APIs (Application Programming Interfaces) provide a way for applications to
+communicate with each other. We already consumed an API earlier in the day:
+**Github API**. We managed to _communicate_ with Github and get important
+information. We - the client - can use this information in a number of different
+ways. Our client, in this case, is a Web page but it could have easily been a
+Mobile Application, or a TV setbox etc...
+
+> **Let's** watch this video about APIs -
+> [What is an API](https://www.youtube.com/watch?v=s7wmiS2mSXY)
+
+> **Exercise**: Let's expose the `posts.json` as an API for other clients to
+> consume through the url `/api/get-posts`. Hint: make use `res.sendFile`
+
+## REST API
```

Comment:
I am not sure of the structure of the whole module .. but discussing REST took us all on a tangent that no one understood last time. Maybe it can remain there but discuss it next class (or just not spend loads of time on it)